src/devices/shelly.ts

```diff
@@ -45,6 +48,313 @@ const shellyModernExtend = {
+            e.text("rpcData", ea.STATE_GET).withLabel("Data").withDescription("RX Data").withCategory("diagnostic"),
+        ];
+        const exposesPowerstripUI: Expose[] = [
+            e
+                .enum("ledMode", ea.STATE_SET, ["off", "switch", "power"])
+                .withLabel("LED Mode")
+                .withDescription("Controls the behaviour of the LED rings around the sockets")
+                .withCategory("config"),
+            e
+                .composite("ledColors", "ledColors", ea.ALL)
+                .withFeature(featurePercentage("onR", "Red (on)"))
+                .withFeature(featurePercentage("onG", "Green (on)"))
+                .withFeature(featurePercentage("onB", "Blue (on)"))
+                .withFeature(featurePercentage("onBrightness", "Brightness (on)"))
+                .withFeature(featurePercentage("offR", "Red (off)"))
+                .withFeature(featurePercentage("offG", "Green (off)"))
+                .withFeature(featurePercentage("offB", "Blue (off)"))
+                .withFeature(featurePercentage("offBrightness", "Brightness (off)"))
+                .withLabel("LED colors in 'switch' mode")
+                .withCategory("config"),
+            featurePercentage("ledPowerBrightness", "LED brightness in 'power' mode").withCategory("config"),
+            e
+                .composite("ledNightMode", "ledNightMode", ea.ALL)
+                .withFeature(e.binary("enable", ea.STATE_SET, true, false))
+                .withFeature(featurePercentage("brightness", "Brightness"))
+                .withFeature(e.text("from", ea.STATE_SET).withLabel("Active from").withDescription("hh:mm"))
+                .withFeature(e.text("until", ea.STATE_SET).withLabel("Active until").withDescription("hh:mm"))
+                .withLabel("LED night mode")
+                .withDescription("Adjust LED brightness during night time")
+                .withCategory("config"),
+            e
+                .composite("buttonsEnabled", "buttonsEnabled", ea.ALL)
+                .withFeature(featureButtonEnabled(0))
+                .withFeature(featureButtonEnabled(1))
+                .withFeature(featureButtonEnabled(2))
+                .withFeature(featureButtonEnabled(3))
+                .withLabel("Buttons enabled")
+                .withCategory("config"),
+        ];
+
+        // biome-ignore lint/suspicious/noExplicitAny: generic
+        const fromZigbee: Fz.Converter<any, any, any>[] = [
+            {
+                cluster: "shellyRPCCluster",
+                type: ["attributeReport", "readResponse"],
+                convert: (model, msg, publish, options, meta) => {
+                    const state: KeyValue = {};
+
+                    // Diagnostic data
+                    if (msg.data.rxCtl !== undefined) {
+                        state.rpcRxCtl = msg.data.rxCtl;
+                        state.rpcData = "";
```

Comment:
```suggestion
                        state.rpc_data = "";
```

(same applies to other attributes)